runtime/databricks/automl_runtime/sklearn/pandas_simple_imputer.py

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2023 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import pandas as pd
+from sklearn.impute import SimpleImputer
+
+class PandasSimpleImputer(SimpleImputer):
+    """
+    A wrapper of `SimpleImputer` with the support for pandas dataframe output.
+    """
+
+    def fit(self, X, y=None):
```

Comment:
Should we add type annotations here? Also on transform

Reply:
Sure.